src/extend/tfoust/index.js

```diff
@@ -0,0 +1,14 @@
+export const extend = async (_chainId, metadata) => {
+    const [series, tokenNumber] = metadata.name.split("#");
+
+    if (tokenNumber && parseInt(tokenNumber) < 100) {
+        metadata.attributes['Token Number'] = 'Double Digits'
```

Comment:
attributes is an array of objects (see an example [here](https://github.com/reservoirprotocol/metadata/blob/main/src/extend/chimpers/index.js)) so this will break the response.

Reply:
thanks - just pushed a fix. lmk if that works!